package/cfg/k3s-cis-1.7-hardened/node.yaml

```diff
@@ -284,40 +276,31 @@ groups:
               set: false
           bin_op: or
         remediation: |
-          If using a Kubelet config file, edit the file to set `makeIPTablesUtilChains` to `true`.
-          If using command line arguments, edit the kubelet service file
-          $kubeletsvc on each worker node and
-          remove the --make-iptables-util-chains argument from the
-          KUBELET_SYSTEM_PODS_ARGS variable.
-          Based on your system, restart the kubelet service. For example:
-          systemctl daemon-reload
-          systemctl restart kubelet.service
+          If using the K3s config file /etc/rancher/k3s/config.yaml, set the following parameter.
+          kubelet-arg:
+          - "make-iptables-util-chains=true"
+          If using the command line, run K3s with --kubelet-arg="make-iptables-util-chains=true".
+          Based on your system, restart the k3s service. For example,
+          systemctl restart k3s.service
```

Comment:
Other areas recommend preceding the `systemctl restart`  with `systemctl daemon-reload`. Any reason why we would not do the `daemon-reload` on this case? Otherwise we should try keeping the same recommendation whenever it applies.

Reply:
If they use the config.yaml option, they don't need a reload of the daemon, because the k3s.service file has not changed. They just need to restart K3s for the new config to be picked up.